docs/specification.md

```diff
@@ -54,10 +54,12 @@ Required `Safelist` entrypoint:
 Additionaly, it would be nice to have the following entrypoint:
 * `checkConstraints ([address, constraint])`.
   * Description: here `constraint` is property that needs to be checked for given
-  address. Can be one of the following:
-  `isWhitelisted`, `isNonBlacklisted`.
-  This entrypoint doesn't do anything in case all constraints are satisfied
-  and fails with an error otherwise.
+    address. Can be one of the following:
+    `isWhitelisted`, `isNonBlacklisted`.
+
+    With Michelson representation is `or unit unit`.
```

Comment:
1. This sentence doesn't sound correct, please check it.
2. From Michelson standpoint this type doesn't look sane. We can make it a bit saner by annotating `unit`s, but what do you think about passing `((list address) :whitelist, (list address) :blacklist))` instead?